### DISCUSSION TITLE
Make soft-wrap support toggle-option

### DISCUSSION THREAD

**Discussion by dryBoneMarrow** [helix-editor/helix#10110] [General] [answered]
`soft-wrap.enable` does not support toggle yet, thus it is not possible to switch between soft wrap enabled and disabled with one keybinding.
It would be nice if the `toggle-option` command would support toggling the `soft-wrap.enable` property.

**Comment by pascalkuthe** [comment, marked as answer]
`:toggle soft-wrap.enable` works just fine

**Reply by dryBoneMarrow** [reply]
I'm very confused right now. Somehow it works even though I could swear that I tried exactly that command, if I would have misspelled the command the error message would be different from the one I got. Probably I'm just stupid. Anyways sorry for wasting your time

**Reply by chtenb** [reply]
For me this toggle doesn't work when soft-wrap is set in languages.toml (`soft-wrap = { enable = true }`). Is that intended?
